src/web/traceViewer/ui/actionList.tsx

```diff
@@ -33,7 +33,27 @@ export const ActionList: React.FC<ActionListProps> = ({
   onSelected = () => {},
   onHighlighted = () => {},
 }) => {
-  return <div className='action-list'>{actions.map(actionEntry => {
+  return <div
+    className='action-list'
+    tabIndex={0}
```

Comment:
The more correct way to do this would be to put tabIndex=0 on the currently selected action, and tabIndex=-1 on all others. But you do you :)